src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -538,6 +543,51 @@ public static Builder builder() {
         return new Builder();
     }
 
+    public void annotateTtl(TaskListener listener) {
+        var kubernetesCloud = getKubernetesCloud();
+        Optional.ofNullable(kubernetesCloud.getGarbageCollection()).ifPresent(gc -> {
+            var ns = getNamespace();
+            var name = getPodName();
+            var l = Instant.now();
+            if (lastRefresh == 0
```

Comment:
Is it really necessary to save `lastRefresh`? Why not just always set it to the current time?

Reply:
I was trying to limit the amount of calls against the api, to cover for large amount of agents to manage.

Reply:
Let's just consider we're not going to need it. I wished there was an API to bulk annotate pods.